src/nvim/os/fs.c

```diff
@@ -192,13 +192,15 @@ int os_open(const char* path, int flags, int mode)
 
 /// Get stat information for a file.
 ///
-/// @return OK on success, FAIL if a failure occurred.
+/// @return `true` on success, `false otherwise.
 static bool os_stat(const char *name, uv_stat_t *statbuf)
   FUNC_ATTR_NONNULL_ALL
 {
   uv_fs_t request;
```

Comment:
No, it doesn't.. so it's overwritten with the uninitialized values in `os_stat`. Also, now that I think of it, won't `= {0}` only initialize the first member of `uv_stat_t`?